pkg/nsx/client.go

```diff
@@ -90,6 +92,7 @@ func GetClient(cf *config.NSXOperatorConfig) *Client {
 	ruleClient := security_policies.NewRulesClient(restConnector(cluster))
 	infraClient := nsx_policy.NewInfraClient(restConnector(cluster))
 	clusterControlPlanesClient := enforcement_points.NewClusterControlPlanesClient(restConnector(cluster))
+	realizedStateClient := realized_state.NewRealizedEntitiesClient(restConnector(cluster))
```

Comment:
I prefer to use `realizedEntitiesClient` as the var name because it will be easier to match the struct in vsphere-automation-sdk-go (there're many lookalike structs, e.g. `NewRealizedEntitiesClient` and `NewRealizedEntitiyClient`.. So we'd better to use the full name to distinguish them more carefully)

Reply:
Make sense, will update the name, thanks.